pkg/registry/apis/provisioning/usage/usage.go

```diff
@@ -0,0 +1,85 @@
+package usage
```

Comment:
no changes in the logic itself only moved things and changed the signature of the function to build the function.